src/include/stir/ProjDataInfo.h

```diff
@@ -150,7 +150,9 @@ class ProjDataInfo
 
   //! Similar to create_shared_clone() but returns a non-tof version of ProjDataInfo setting tof mashing factor = 0
   inline shared_ptr<ProjDataInfo> create_non_tof_clone() const;
-
+  //! Similar to create_non_tof_clone() but returns a single TOF version of the ProjDataInfo setting the TOF mashing factor sush that,
+  //! a single TOF bin remains. This is usefull to apply the coincidence window boundaries without affecting the probabilities.
+    inline shared_ptr<ProjDataInfo> create_single_tof_clone() const;
```

Comment:
I suppose we need this for backwards compatibility? i.e. need to keep `create_non_tof_clone` unmodified? If so, I'd be tempted to add a bool to `create_non_tof_clone()` instead.

Reply:
Note: will be obsolete once we merge another PR on SSRB for TOF